TextilgallerianKuponger/Domain/Entities/BuyProductXRecieveProductY.cs

```diff
@@ -21,7 +21,7 @@ public class BuyProductXRecieveProductY : ProductCoupon
         /// </summary>
         public override bool IsValidFor(Cart cart)
         {
-            throw new NotImplementedException();
+            return base.IsValidFor(cart);
```

Comment:
Haha nä, men det ska in mer logik där för vi var tvungna att få testen att fungera bara med föräldern så ska dit fler tester och mer logik allt eftersom då.